client/js/components/month/month-week-board.js

```diff
@@ -0,0 +1,20 @@
+/** @jsx React.DOM */
+var React = require('react');
+var MonthEvent = require('./month-event.js');
+var _ = require('underscore');
+
+var MonthWeek = React.createClass({
```

Comment:
Same comment as above. With this file named month-week-board.js, I would expect the exported value to be:

```var MonthWeekBoard = React.createClass({.....```